### ISSUE TITLE
Battery Critical error at 100 percent 

### ISSUE THREAD

**Issue by Ravieroy** [ubports/ubuntu-touch#1949]
- Device: Xiaomi Mi A2
- Channel: Stable 
- Build Details: 
   - OS Build number : OTA-22
   - UBportsImage part: 20220131-6087
   - Device Build Description: lineage_halium_arm64-userdebug 9 PQ#A.190801.002 423 test-keys

### Steps to reproduce

The Bug appears when the phone is fully charged and still connected to the power outlet. 

### Expected behavior

The expected behavior is that charging should stop and notification should pop that battery is fully charged. Instead, a critical battery error pops up. 

### Actual behavior

Well, this is a non-lethal simple little bug and should be easy to reproduce. At 100 percent, there is a notification of critical battery and 0% charge remaining. (see the images)

![img_1 (1)](https://user-images.githubusercontent.com/81288438/156933129-62e88e52-1d56-4a27-aa91-3f382eba4961.jpg)

![img_2 (1)](https://user-images.githubusercontent.com/81288438/156933131-d620d078-13ce-4d5b-9105-e84ff306b69e.jpg)

![img_3 (1)](https://user-images.githubusercontent.com/81288438/156933132-4dae2f8e-86d0-4cf4-b3a1-f563462efe14.jpg)

Thanks for all your effort.

**Comment by Flohack74** [comment]
Yes battery management is wonky since Halium 9. We will try to tackle this soon. But its only a display bug: Charging works normal, the kernel and drivers take care of it.
